React/CoreModules/RCTDevSettings.mm

```diff
@@ -502,6 +509,8 @@ @interface RCTDevSettings () <NativeDevSettingsSpec>
 
 @implementation RCTDevSettings
 
+RCT_EXPORT_MODULE()	// TODO(macOS ISS#2323203)
```

Comment:
This being missing seems a merge failure when I pulled in code from upstream, as [it exists there](https://github.com/facebook/react-native/blob/0.63-stable/React/CoreModules/RCTDevSettings.mm#L131). In that case the comment isn’t necessary.

Reply:
ah, I see. I'll remove this if I touch this file in the near future.